docs/content/controls/_select2.md

```diff
@@ -0,0 +1,50 @@
+#Select2
+A select field based on the select2 plugin. Accepts an `array` of `options` in which the `key` is the value and the `value` is the option name. Set `multiple` to `true`, to allow multiple value selection choice.
+
+*Returns* `string|array` The selected option key, or an array of selected values if `multiple == true`
+
+##Example
+
+```php
+$this->add_control(
+  'show_elements',
+  [
+     'label'       => __( 'Show Elements', 'your-plugin' ),
+     'type' => Controls_Manager::SELECT2,
+     'options' => [
+     	'title'       => __( 'Title', 'your-plugin' ),
+     	'desctiption' => __( 'Desctiption', 'your-plugin' ),
+     	'button'      => __( 'Button', 'your-plugin' ),
+     ],
+	 'multiple' => 'true',
```

Comment:
right, but what `'multiple' => 'false'` would do in javascript? 

In this context, to be more specific:

```
<# var multiple = ( data.multiple ) ? 'multiple' : ''; #>
```